Cargo.toml

```diff
@@ -9,27 +9,37 @@ homepage= "https://github.com/casbin-rs/sqlx-adapter"
 readme= "README.md"
 
 [dependencies]
-casbin = { version = "2.0.2", default-features = false }
-sqlx = {version = "0.4.0-beta.1", default-features = false, features = [ "macros" ]}
-async-trait = "0.1.40"
+# Pin <2.0.3 to force tokio 0.2.xx for sqlx compatibility
+casbin = { version = "<2.0.3", default-features = false }
```

Comment:
Is this compability problem still existing?

Reply:
The compatibility issue will be solved by this pull request, for now it still exists:

https://github.com/launchbadge/sqlx/pull/983

Reply:
Thanks, so we just wait it fixing.

Reply:
Wait, did you mean the SQLx or the Casbin version? For Casbin I saw that the 2.0.5 version has tokio 0.2 so it can be changed

Reply:
My bad yeah, I misread the "comment on lines", in the latest commit I fixed the casbin version to the latest

Reply:
I think this conversation is done.